dask/bytes/core.py

```diff
@@ -39,6 +39,8 @@ def write_block_to_file(data, f, compression, encoding):
         encoding and binary mode.
     """
     original = False
+    opener, path, kwargs = f
+    f = opener(path, **kwargs)
```

Comment:
Ah, so now we're not passing in a file, we're passing in enough to make the file locally.